api/v2.0/swagger.yaml

```diff
@@ -1755,6 +1755,59 @@ paths:
           $ref: '#/responses/401'
         '500':
           $ref: '#/responses/500'
+  /auditlog-exts:
+    get:
+      summary: Get recent logs of the projects which the user is a member of
```

Comment:
1. As per the design the user has to be a project admin to see the audit log of a project.
2. I don't really think we need to provide a aggregated view of audit log of different projects.  How about we use this API only for system admin?